core/Inst.hpp

```diff
@@ -372,8 +433,17 @@ namespace olympia
         const bool is_condbranch_;
         const bool is_call_;
         const bool is_csr_;
+        const bool is_vector_;
         const bool is_return_;
+        bool has_uops_;
+        bool has_tail_; // Does this vector instruction have a tail?
+        uint64_t uop_done_count_ = 1; // start at 1 because the uop count includes the parent instruction
+        uint64_t uop_count_ = 0;
+        VCSRs VCSRs_;
 
+        bool is_blocking_vset_ = false;
```

Comment:
Would be good to comment what a blocking vset is.

Reply:
Addressed in [73ca9f4](https://github.com/riscv-software-src/riscv-perf-model/pull/167/commits/73ca9f4adb0bce0b8756a9b1ab3d88bf96c589bd)